docs/additional-features/stress-testing.md

```diff
@@ -9,13 +9,14 @@ This guide assumes that:
 
 - You have a working polygon-sdk network up and running (using IBFT or dev)
 - Both your JSON-RPC and gRPC endpoints are reachable
```

Comment:
@zivkovicmilos I guess we dont need gRPC.

Reply:
Good catch, we relied on gRPC in an earlier version of the implementation 💯 

Fixed 👍 

c64b209